nifi-nar-bundles/nifi-slack-bundle/nifi-slack-processors/pom.xml

```diff
@@ -146,6 +142,11 @@
             <artifactId>nifi-proxy-configuration-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-api</artifactId>
+            <scope>compile</scope>
```

Comment:
Is there a reason for this change? The `nifi-api` should be provided.

Reply:
Nope, not sure how that happened :)